recipe/meta.yaml

```
@@ -12,34 +12,43 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - wheel
+    - setuptools
   run:
     - azure-core <2.0.0,>=1.23.0
     - cryptography >=2.5
     - msal <2.0.0,>=1.24.0
     - msal_extensions <2.0.0,>=0.3.0
-    - python >=3.7
+    - python
 
 test:
   imports:
     - azure.identity
     - azure.identity._credentials
-  # commands:
-  #   - pip check
-  # requires:
-  #   - pip
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity
   summary: Microsoft Azure Identity Library for Python
+  description: |
+    The Azure Identity library provides Microsoft Entra ID (formerly Azure Active Directory) token
+    authentication support across the Azure SDK. It provides a set of TokenCredential implementations,
+    which can be used to construct Azure SDK clients that support Microsoft Entra token authentication.
   license: MIT
+  license_family: MIT
   license_file: LICENSE.txt
```

```suggestion
  license_file: LICENSE
```
(The archive contains LICENSE, LICENSE.txt is from the feedstock)

---

Nice catch.